src/js/search-bar/search-bar.view.js

```diff
@@ -4,26 +4,44 @@ import {selectDomElement} from "../utils/dom-selector";
 import {SearchBarEvent} from "./search-bar.constants";
 
 const template = `
-    <button id="awesome-button">click</button>
-`
+    <div class="search-bar hover-initial-search-bar" id="search-bar">
+        <input class="search-bar__input" id="search-input" type="search" placeholder="Search...">
+        <button class ="search-bar__icon" id="search-btn">
+            <img src="https://i.ibb.co/d5rcTzD/icons8-magnifying-glass-64.png" alt="search-icon" class ="search-bar__img">
+        </button>
+    <div class="search-bar__overlay" id="search-bar-overlay"></div>
```

Comment:
there is no need to create separate dom-element only for toggling search bar. we may have only flag (true \ false field) in view class, where we deside to send search event or just toggle the search-bar